src/elona/lua_env/mod_serializer.cpp

```diff
@@ -0,0 +1,52 @@
+#include "mod_serializer.hpp"
+#include "../character.hpp"
+#include "../item.hpp"
+
+namespace elona
+{
+namespace lua
+{
+
+std::pair<int, int> get_start_end_indices(
+    const std::string& kind,
+    ModInfo::StoreType store_type)
+{
```

Comment:
If `store_type` is `global`, it returns `[start, end]` (inclusive), but otherwise, it returns `[start, end)` (exclusive). The caller treats the result as exclusive range, so ` - 1` looks unnecessary.